src/OmniSharp.Roslyn.CSharp/Helpers/QuickFixExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Models;
+
+namespace OmniSharp.Helpers
+{
+    internal static class QuickFixExtensions
+    {
+        internal static async Task AddQuickFix(this ICollection<QuickFix> quickFixes, OmniSharpWorkspace workspace, Location location)
```

Comment:
You *could* just call this `Add` since it's going onto an `ICollection<QuickFix>`.